.golangci.yml

```diff
@@ -1,28 +1,32 @@
+# See https://golangci-lint.run/usage/configuration/#config-file
```

Comment:
I assume this change is required because you can't use too old version of go with the new golangci-lint tool.

I am reluctant to do this change. This will cause potentially avoidable conflicts in the future.

Reply:
kept the file as-is